org-formation/700-aws-sso/_tasks.yaml

```diff
@@ -2218,3 +2222,21 @@ SsoOpenchallengesProdAdmin:
     instanceArn: !Ref instanceArn
     principalId: !Ref OpenchallengesProdAdminGroup
     permissionSetArn: !CopyValue [ !Sub '${resourcePrefix}-${appName}-admin-permission-set-arn' ]
+
+
+SsoCodeOceanProdAdmin:
+  Type: update-stacks
+  DependsOn: SsoAdministrator
+  Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.2.11/templates/SSO/aws-sso.yaml
+  StackName: !Sub '${resourcePrefix}-${appName}-codeocean-prod-admin'
+  StackDescription: 'SSO: admin role used by CodeOcean prod admin group'
+  DefaultOrganizationBindingRegion: !Ref primaryRegion
+  DefaultOrganizationBinding:
+    IncludeMasterAccount: true
+  OrganizationBindings:
+    TargetBinding:
+      Account: !Ref OpenChallengesProdAccount
```

Comment:
You missed updating this to CodeOceanProdAccount

Reply:
good catch, fixed now.